packages/pmml-editor/src/editor/reducers/DelegatingModelReducer.ts

```diff
@@ -16,38 +16,63 @@
 import { HistoryAwareModelReducer, HistoryService, ModelReducerBinding } from "../history";
 import { Model } from "@kogito-tooling/pmml-editor-marshaller";
 import { Reducer } from "react";
-import { AllScorecardActions } from "./ScorecardReducer";
-import { getModelType, ModelType } from "..";
+import { getModelType, ModelType } from "../PMMLModelHelper";
+import { AllActions } from "./Actions";
+import get = Reflect.get;
 
-export const DelegatingModelReducer: HistoryAwareModelReducer<AllScorecardActions> = (
+const reduce = (model: Model, action: AllActions, reducers: Map<ModelType, ModelReducerBinding<any, any>>) => {
+  const modelType = getModelType(model);
+  const reducer = reducers.get(modelType);
+  if (reducer !== undefined) {
+    return reducer.factory(reducer.reducer(model, action));
+  }
+  return model;
```

Comment:
```suggestion
    return reducer?.factory(reducer.reducer(model, action)) ?? model;
```

Does it even work? If so, I think it's nicer 😝

Reply:
It works, it is nicer, it is included in a new commit.